nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentJsonWriter.java

```diff
@@ -78,4 +95,13 @@ protected static void writeCommentEntity(Comment entity, JsonGenerator jg) throw
             jg.writeStringField(EXTERNAL_ENTITY, ((ExternalEntity) entity).getEntity());
         }
     }
+
+    protected void writeRepliesSummary(Comment entity, JsonGenerator jg) throws IOException {
+        CoreSession session = ctx.getSession(null).getSession();
+        PartialList<Comment> comments = commentManager.getComments(session, entity.getId(), 1L, 0L, false);
```

Comment:
Not sure if it's the right place to do it. Do you find another place where we did it ?
Let's discuss it with @efge and/or @troger

Reply:
Just found strange to call a service from a writer. Is it something we want ?